carbon-design-system/carbon
# Add Typescript annotations to SelectSkeleton

Closes #12556 

Add Typescript annotations to SelectSkeleton.

#### Changelog

**Changed**

- SelectSkeleton.js has been converted to tsx

#### Testing / Reviewing

- Run yarn build; yarn storybook in packages/react. Confirm that the Select > Skeleton story works as expected.
- Add SelectSkeleton element to bottom of SelectSkeleton.tsx and confirm it compiles. Add className="foo" and confirm it compiles. Change to className={3} and confirm you receive a Typescript error.
